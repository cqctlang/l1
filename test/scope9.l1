@global main;
{
	@defloc foo()
	{
		printf("%a\n", unboundvar);
	}

	@define main()
	{
		gc(); gc(); gc(); gc(); gc(); gc();
		foo();	
	}
}

main();
