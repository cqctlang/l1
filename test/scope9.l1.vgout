scope9.l1:5: warning: reference to unbound variable: unboundvar
error: reference to unbound variable: unboundvar
                 foo	(scope9.l1:5)
               entry	(scope9.l1:15)
