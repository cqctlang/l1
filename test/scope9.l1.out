scope9.l1:5: warning: reference to unbound variable: unboundvar
entry: var params, 2 locals, 0 temps
foo: 0 params, 0 locals, 0 temps
main: 0 params, 0 locals, 0 temps
error: reference to unbound variable: unboundvar
                 foo	(scope9.l1:5)
               entry	(scope9.l1:15)
